src/main/resources/Lang/uk-lang.yml

```diff
@@ -0,0 +1,44 @@
+# All messages support Minecraft (&f) and hex (#ffffff) color codes.
```

Comment:
What will the difference be between this lang file vs something like an EN_US lang?